pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/metrics/LocMetric.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.oom.metrics;
+
+import net.sourceforge.pmd.lang.java.ast.ASTClassOrInterfaceDeclaration;
+import net.sourceforge.pmd.lang.java.ast.ASTMethodOrConstructorDeclaration;
+import net.sourceforge.pmd.lang.java.oom.ClassMetric;
+import net.sourceforge.pmd.lang.java.oom.OperationMetric;
+import net.sourceforge.pmd.lang.java.oom.visitor.PackageStats;
+
+/**
+ * Lines Of Code metric. Simply equates the length in lines of code of the measured entity.
+ *
+ * @author Clément Fournier
+ */
+public class LocMetric extends AbstractMetric implements ClassMetric, OperationMetric {
+
+    @Override
+    public double computeFor(ASTClassOrInterfaceDeclaration node, PackageStats holder) {
+        return node.getEndLine() - node.getBeginLine();
```

Comment:
I didn't put much thought into it lol

I guess we could have both metrics, one version that counts with comments and the other not, and set which implementation will be used by rules that use LOC via a rule property.